ocaml/xapi/storage_migrate.ml

```diff
@@ -269,7 +271,7 @@ let tapdisk_of_attach_info (backend:Storage_interface.backend) =
 let with_activated_disk ~dbg ~sr ~vdi ~dp f =
   let attached_vdi =
     Opt.map (fun vdi ->
-        let backend = Local.VDI.attach2 dbg dp sr vdi false in
```

Comment:
Instead of empty string I think we shoul duse the actual domid. If this is something attached to Dom0 then the ID should be 0.